lib/kitchen/directions/bake_toc.rb

```diff
@@ -88,6 +88,11 @@ def self.li_for_page(page)
           number.prepend(child: part_text.paste)
         end
 
+        # Targets the Answer Key
+        if page.has_ancestor?('composite_chapter') && page.ancestor('composite_chapter').search('span').first.text == 'Answer Key'
```

Comment:
Is there a better way to get at the correct pages?

Reply:
let's look at this together